src/python/espressomd/visualization_opengl.py

```diff
@@ -2356,7 +2354,9 @@ def draw(self):
 
 # OPENGL DRAW WRAPPERS
 
-def set_solid_material(color, material=[0.6, 1.0, 0.1, 0.4, 1.0]):
+def set_solid_material(color, material=None):
```

Comment:
Why this change? This pylint warning is fixed by transforming the list into a tuple.

Reply:
My bad, a rather obvious solution &hellip;